applications/FluidDynamicsApplication/tests/TwoFluidMassConservationProcTest/2Dtest.json

```diff
@@ -17,65 +17,66 @@
         "material_import_settings"       	: {
```

Comment:
If I'm not wrong, this is the old process test. Why do you need to change it? As far as I see all these changes are aesthetic. If so it is, please remove as this makes the revision blurry.

Reply:
Only a new line needs to be added in order to deactivate mass source term. I removed other changes.